ui/src/components/issuers/Onboarding.tsx

```diff
@@ -0,0 +1,138 @@
+import { Avatar, Card, Divider, Flex, Grid, Typography, message } from "antd";
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { createIssuer } from "../../adapters/api/issuers";
+import { IssuerFormData } from "src/adapters/parsers/view";
+import IconCheck from "src/assets/icons/check.svg?react";
+import IconIssue from "src/assets/icons/credential-card.svg?react";
+import IconSchema from "src/assets/icons/file-search-02.svg?react";
+import IconIdentity from "src/assets/icons/fingerprint-02.svg?react";
+
+import { IssuerForm } from "src/components/issuers/IssuerForm";
+import { useEnvContext } from "src/contexts/Env";
+
+import { useIssuerContext } from "src/contexts/Issuer";
+import { ROUTES } from "src/routes";
+
+import { FINALISE_SETUP } from "src/utils/constants";
+
+const cards = [
+  {
+    icon: <IconIssue />,
```

Comment:
Yes, I agree with you, but it's a global styling, and a lot of colors now look wrong, I discussed  that I would check and refactor all colors, but it will be after I complete this part of the work

that's why this page a did using old colors scheme